composer.json

```diff
@@ -16,11 +16,7 @@
         },
         {
             "type": "vcs",
-            "url": "git@github.com:previousnext/drush_cmi_tools.git"
-        },
-        {
-            "type": "vcs",
-            "url": "git@github.com:mikeyp/google_tag.git"
+            "url": "git@github.com:comicrelief/behat-tools.git"
```

Comment:
@Saphyel can we review that we don't need drush_cmi_tools nor google_tag (replaced with an official d.o. module)?

Reply:
We don't get the modules and having the repositories adds an extra call to github but we actually do nothing with them  also is causing issues to @Heleen-cr , so removing the repositories is an improvement

If we want back `drush_cmi_tools` to this profile we should create an issue and add again the repository.